internal/inspect/detectors/quarto.go

```diff
@@ -28,6 +28,10 @@ var specialYmlFiles = []string{
 	"_brand.yaml",
 }
 
+// When there is no quarto binary or version detected,
+// this is the default we will use on deployment configurations.
+var defaultQuartoVersion = "1.7.34"
```

Comment:
@jonkeane @dotNomad Thoughts on this? 

Since there is no Quarto binary to check the version, I'm adding a fallback because the deployment configuration is not valid without a version and we also need it to generate the manifest.

Reply:
My understanding of this is the `[quarto]` section requires a version unlike the `[python]` and `[r]` sections can use a selected/installed interpreter. With that being the case I'm fine with this hard code for now as well.